dialect_redshift.go

```diff
@@ -0,0 +1,19 @@
+package gorp
+
+type RedshiftDialect struct {
+	PostgresDialect
+}
+
+func (d RedshiftDialect) InsertAutoIncrToTarget(exec SqlExecutor, insertSql string, target interface{}, params ...interface{}) error {
+	rows, err := exec.Query(insertSql, params...)
+	defer rows.Close()
+	return err
+}
+
+func (d RedshiftDialect) AutoIncrBindValue() string {
+	return ""
```

Comment:
should we include a code comment here to remind ourselves that redshift forbids setting identity columns?

Reply:
9604ea7